website/content/docs/drivers/remote/ecs.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: 'Task Driver Plugins: ECS Task Driver'
+description: >-
+  The AWS ECS Task Driver is an example Remote Task Driver.
+---
+
+# ECS Task Driver
+
+Name: `nomad-driver-ecs`
+
+Homepage: https://github.com/hashicorp/nomad-driver-ecs
+
+~> **Note:** The ECS Task Driver is an example Remote Task Driver and **not
+intended for production use.**
+
+The ECS task driver plugin for Nomad allows running [AWS ECS][ecs] tasks via
+Nomad. Allocations for these jobs are scheduled onto Nomad clients like
+traditional task drivers, however the actual task code in executed remotely in
+AWS ECS. The Nomad client agent manages the remote ECS task like any other
+local Nomad task: restarting it if it fails, stopping it when requested, etc.
+
+When a Nomad node assigned allocations with ECS tasks is drained, the ECS tasks
+are *not stopped.* Instead the replacement allocations reconnect to the
+original ECS tasks to avoid unnecessary downtime.
+
+If a Nomad node assigned allocations with ECS tasks crashes and is considered
+`down`, the replacement allocations for the `lost` allocations reconnect to the
+original ECS tasks to avoid unnecessary downtime. If the original crashed Nomad
+node restarts, it will detect `lost` allocations and stop monitoring them since
+a new node has taken over.
+
+## Client Requirements
+
+The AWS ECS Task Driver is not currently built into Nomad and must be
+[downloaded][download] onto the client host in the configured [plugin
```

Comment:
note that the releases link is empty - https://github.com/hashicorp/nomad-driver-ecs/releases

Reply:
We'll actually be using releases.hashicorp.com for these builds, and I'll get those uploaded tomorrow.